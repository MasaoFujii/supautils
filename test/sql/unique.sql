select supa.unique(array[1, 2, 2, 4])
