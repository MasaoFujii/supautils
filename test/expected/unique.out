select supa.unique(array[1, 2, 2, 4])
 unique  
---------
 {1,2,4}
(1 row)

